login.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+      integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+<title>Register</title>
+
+   <link href="styles.css" rel="stylesheet">
+
+  </head>
+  <body>
+    <div class="circle">
+  <div class="push-btn-container">
+<!--     <div class="centering"> -->
+    <div class="push-toggle">
+        <input onclick="myFunction()" type="checkbox">
+        <span class="push-button"></span>
+        <span class="push-btn-label">
+          <i class="far fa-lightbulb"></i>
+          </span>
+      </div>
+    </div>
+  </div>
+</div>
+ <div class="container">
+     <div class="form">
+       <div class="heading">
+
+       </div>
+       <div class="main">
+         <div class="register">
+           <form id="register" method="post">
+           <label for="username">Name of the Project : </label>

```

Comment:
I don't think this is the page we want, there is nothing about login. @deek121477 did you tested your changes locally??
![image](https://user-images.githubusercontent.com/74113200/111108169-50a07a80-857e-11eb-9e71-98a9454b6942.png)

Reply:
The toggle is also not working